### PR TITLE
Alert

### PR DESCRIPTION
**Pull request by kr-sarvesh**
https://compassionate-hopper-6186fd.netlify.app/components/alert/alert.html

please review by alert component

### CODE REVIEW

**Review by rahul31agrawal — APPROVED** [review]
looks good to merge

File: components/alert/alert.css [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+
+.wrapper {
+  width: 100%;
+  min-width: 100vh;
+  height: max-content;
+  margin: auto;
+}
+
+.wrapper .bottom-nav {
+  width: 100%;
```
Comment (sakshi006) [inline components/alert/alert.css:15]: I think for the styles of common components like navbar, you should put them in the common stylesheet. ELse, you will have to add this code to every component's CSS file.

File: components/alert/alert.css [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+/* media queries starts from here */
+
+@media screen and (max-width: 900px) {
+  #navbar {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    margin: auto;
+    align-items: center;
+  }
```
Comment (sakshi006) [inline components/alert/alert.css:109]: This is also common style for all pages i guess.

**Review by sakshi006 — COMMENTED** [review]
Made some comments, please check.

**Pull request merged**